Sources/MessagingPushAPN/autogenerated/AutoDependencyInjection.generated.swift

```diff
@@ -54,9 +54,7 @@ extension DIGraph {
     // internal scope so each module can provide their own version of the function with the same name.
     @available(iOSApplicationExtension, unavailable) // some properties could be unavailable to app extensions so this function must also.
     func testDependenciesAbleToResolve() -> Int {
-        var countDependenciesResolved = 0
-
-        return countDependenciesResolved
+        return 0
```

Comment:
makes sense, but it's an autogenerated file, so this will be reverted probably when the next time `make generator` runs.

Reply:
Correct. That's why the CI status check is failing on this PR at the moment. We do a `git diff` on the CI to make sure that `make format` was run *before* pushing the code to the remote git repo.

Reply:
ah, yes. That is very true. Hard to differentiate till you look at the file path :D